packages/driver-interface/src/adapters/amqp.ts

```diff
@@ -13,4 +13,5 @@ export const queues = {
   OFFER_BOOKING_TO_TELEGRAM_VEHICLE: 'offer_bookings_to_telegram_vehicles',
   ADD_BOOKING_INFO: 'add_booking_info_in_driver_interface',
   ADD_INSTRUCTIONS_TO_VEHICLE: 'add_instructions_to_vehicle',
+  ADD_VEHICLE: 'driver_interface.add_vehicle'
```

Comment:
Should we prepend or append app name? All other appends in some way and also, answers here suggests appending:
https://stackoverflow.com/questions/32220312/rabbitmq-amqp-best-practice-queue-topic-design-in-a-microservice-architecture